ssl/statem/statem_dtls.c

```diff
@@ -800,17 +800,19 @@ static int dtls1_process_out_of_seq_message(SSL_CONNECTION *s,
 static int dtls_get_reassembled_message(SSL_CONNECTION *s, int *errtype,
                                         size_t *len)
 {
-    unsigned char wire[DTLS1_HM_HEADER_LENGTH];
     size_t mlen, frag_off, frag_len;
     int i, ret;
     uint8_t recvd_type;
     struct hm_header_st msg_hdr;
     size_t readbytes;
     SSL *ssl = SSL_CONNECTION_GET_SSL(s);
     int chretran = 0;
+    unsigned char *p;
```

Comment:
Having single letter variable name is discouraged or considered as bad practice. Can we please have a better intuitive and descriptive name?

Reply:
`p` as a memory buffer pointer is IMO OK. What would be the better name? Also the variable name was used below already and changing it means changing the follow up code.

Reply:
I agree that an appropriate name would be beneficial, but I chose to use the same name as in `tls_get_message_header()` for consistency.
Please let me know if you have a naming suggestion, then I'll apply it here and in `tls_get_message_header()`.

Reply:
> Having single letter variable name is discouraged or considered as bad practice. Can we please have a better intuitive and descriptive name?

In general yes I agree - good point. In this case though we widely use `p` for such a purpose in the codebase so I think its meaning is well understood. Also, as @t8m points out, this is really just changing the scope of a pre-existing variable. So I think it is ok here. OTOH, I'm not going to object to a more descriptive name!